.devcontainer/Dockerfile

```diff
@@ -53,65 +102,37 @@ RUN apt-get update -qq && \
     sudo \
     libpng-dev \
     libgomp1 \
-    python3-venv \
     file \
     libmagic-mgc \
     libmagic1 \
     software-properties-common \
     curl \
-    bash-completion && \
-    rm -rf /var/lib/apt/lists/*
-
-###########################
-# Install pip requirements
-###########################
-RUN pip3 install --upgrade pip==24.1
-RUN pip3 install torch torchvision timm opencv-python transforms3d pyopengl setuptools==58.2.0 albumentations --no-binary albumentations,qudida
-
-################
-# STEREOLABS ZED
-################
-# comment out the following line on non-GPU platforms
-RUN wget -q -O ZED_SDK_Linux_Ubuntu22.run https://download.stereolabs.com/zedsdk/4.1/cu121/ubuntu22 && \
-    chmod +x ZED_SDK_Linux_Ubuntu22.run ; ./ZED_SDK_Linux_Ubuntu22.run silent skip_cuda && \
-    ln -sf /lib/x86_64-linux-gnu/libusb-1.0.so.0 /usr/lib/x86_64-linux-gnu/libusb-1.0.so && \
-    rm ZED_SDK_Linux_Ubuntu22.run && \
-    rm -rf /var/lib/apt/lists/*
-RUN mkdir -p /root/Documents/ZED/
-
-######
-# ROS 
-######
-RUN apt-get update -q && add-apt-repository universe && \
-    curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg && \
-    echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | tee /etc/apt/sources.list.d/ros2.list > /dev/null && \
-    apt-get update -q && apt-get install --no-install-recommends -y \
-    ros-dev-tools \
-    ros-${ROS_DISTRO}-desktop \
-    ros-${ROS_DISTRO}-grid-map-rviz-plugin \
-    ros-${ROS_DISTRO}-ament-cmake-clang-format \
-    ros-${ROS_DISTRO}-xacro \
-    ros-${ROS_DISTRO}-nmea-msgs \
-    ros-${ROS_DISTRO}-geographic-msgs \
-    ros-${ROS_DISTRO}-diagnostic-updater \
-    ros-${ROS_DISTRO}-robot-localization \
-    ros-${ROS_DISTRO}-common-interfaces \
-    ros-${ROS_DISTRO}-rmw-cyclonedds-cpp && \
-    rm -rf /var/lib/apt/lists/*
-
-ENV RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
+    bash-completion \
+    && rm -rf /var/lib/apt/lists/*
 
+# bootstrap rmw_cyclonedds_cpp and rosdep 
+# ENV RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
 RUN rosdep init && rosdep update --rosdistro $ROS_DISTRO
```

Comment:
How does this use `uv`? 

Rosdep uses `apt` or `rpm` for system packages, and `pip3` directly for python dependencies. 

Do the python dependencies end up in the system-wide installation of python?

Reply:
"uv environment using pip packages, based on the system package rosdep. This approach keeps dependencies in one place within the uv environment." see [58ba0e6](https://github.com/sense-base/base/pull/38/commits/58ba0e61ae8e799dc48a6c89aca29a3dbc8e5fd3)